services/wfm/src/main/java/org/openkilda/wfm/topology/floodlightrouter/service/MessageSender.java

```diff
@@ -19,9 +19,13 @@
 
 
 public interface MessageSender {
-    void send(Message message, String outputStream, boolean lookupKey);
+    void emitSpeakerMessage(Message message, String region);
 
-    void send(Object payload, String outputStream);
+    void emitSpeakerMessage(String key, Message message, String region);
 
-    void send(String key, Message message, String outputStream);
+    void emitConsumerMessage(Message message);
```

Comment:
Consumer here is not clear tbh, from router perspective both FL and rest of the Kilda could be consumers